lib/install/bin/webpack-dev-server.tt

```diff
@@ -47,9 +53,10 @@ rescue Errno::EADDRINUSE
 end
 
 # Delete supplied host and port CLI arguments
-["--host", "--port"].each do |arg|
+["--host", "--port", "--listen-host"].each do |arg|
   ARGV.delete(args(arg))
   ARGV.delete(arg)
+  ARGV.delete(arg)
```

Comment:
Why did you add this line?  Isn't it redundant with the one above it?

Reply:
haha, don't know - removed it 👍